Add "master device only" editions for the Sunrise IDE kernel ROMs

Konamiman opened the pull request:
This pull request modifies `source/kernel/Makefile`, `source/kernel/drivers/SunriseIDE/driver.mac` and `source/kernel/drivers/SunriseIDE/sunride.asm` so that two additional ROM files are generated:

* Nextor-(version).SunriseIDE.MasterOnly.ROM
* Nextor-(version).SunriseIDE.MasterOnly.emulators.ROM

These versions work as the existing ones, except that slave device detection will be skipped. Users of Carnivore2 should use Nextor-(version).SunriseIDE.MasterOnly.ROM.

Additionally, the temporary work area of the `sunride.asm` driver is changed from C000h to C400h, [to prevent conflicts with the Panasonic FS machines](https://github.com/Konamiman/Nextor/pull/105#issuecomment-1174469950).

Konamiman committed:
Add "master only" editions for the Sunrise IDE ROMs.

Makefile and driver sources have been modified so that now two
additional kernels will be generated:
Nextor-(version).SunriseIDE.MasterOnly.ROM and
Nextor-(version).SunriseIDE.MasterOnly.emulators.ROM

uniabis commented:
Better to use 0C400H for TEMP_WORK of sunride.asm.

Firmware of Panasonic FS-A1 uses 0CBD8h.
Firmware of Panasonic FS-A1F uses 0C3CEh.
Firmware of Panasonic FS-A1MK2 uses 0C3D2h.
Firmware of Panasonic FS-A1ST/WX uses 0C010h-0C015h.

https://msx.org/forum/msx-talk/development/routine-to-bypass-a-firmware?page=2

Wierzbowsky commented:
Works fine in Carnivore2 with v2.50 firmware.

Konamiman committed:
Change temp work area in sunride.asm from C000h to C400h.

This is to prevent conflicts with some Panasonic FS machines
that use C000-C400 as temporary work area at boot time.
